docs/integrations/sessionstack.rst

```diff
@@ -0,0 +1,64 @@
+SessionStack
+============
+
+SessionStack error session replay plugin for Sentry
+```````````````````````````````````````````````````
+
+To integrate the SessionStack player within your Sentry error reports, follow the steps below:
+Add your web app to your SessionStack account
+
+.. image:: http://blog.sessionstack.com/wp-content/uploads/2017/04/create_website_opened.png
```

Comment:
This actually won't work since it's an http asset being served on an https asset. We need to vendor it.